SetSingleScreenVideoMode.brs

```diff
@@ -3,16 +3,25 @@
 ' Plugin return type - As String
 ' See BrightSign docs for a list of supported video modes - https://brightsign.atlassian.net/wiki/spaces/DOC/pages/370676833/Supported+Video+Modes
 
+' Note: This plugin assumes it is executed on BrightSign Model with a single HDMI output. 
+' This will NOT work on a BrightSign Model with multiple HDMI outputs. 
+
 Function setvideomode(videoModeInputs, bsp) As String
 
-	print "setvideomode_ForSingleScreen"
+	print "video mode plugin: setvideomode()"
+
+	' To customize the options of a supported video mode, use the format of the following return 
+  ' statement with the customized options.
```

Comment:
The customized options description here sounds very close to the next block custom modelines. Can we avoid use the word "custom" or "customized" here to better distinguish between them?
e.g. set BrightSign video mode